packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -48,11 +49,28 @@ export function disableCLITelemetry() {
   );
 }
 
+async function getConfigAggregator(): Promise<ConfigAggregator> {
```

Comment:
Note to move this when we can

Reply:
Do we have a ticket to address moving this to a common location?

Reply:
Follow up WI: [W-11528403](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE0000130l0YYAQ/view)